### PR TITLE
update of readme

### PR DESCRIPTION
**Pull request by pavertomato**
[#21] update of readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -244,3 +244,6 @@ docker push ${TAG}
 ```
 
 Instructions on how to run the Import Adapter Docker image locally are available on the [wiki](https://github.com/GoogleCloudPlatform/healthcare-dicom-dicomweb-adapter/wiki/Running-Docker-image-locally).
+
+### Data protection toolkit deployment
```
Comment (danielbeaudreau) [inline README.md]: Please capitalize the title "Data Protection Toolkit Deployment"

File: README.md [hunk 2]
```diff
@@ -244,3 +244,6 @@ docker push ${TAG}
 ```
 
 Instructions on how to run the Import Adapter Docker image locally are available on the [wiki](https://github.com/GoogleCloudPlatform/healthcare-dicom-dicomweb-adapter/wiki/Running-Docker-image-locally).
+
+### Data protection toolkit deployment 
+The adapters can be deployed as part of the [data protection toolkit](https://github.com/GoogleCloudPlatform/healthcare/tree/master/deploy) as gke_workload resource. Sample configuration may be found in this [folder.](https://github.com/GoogleCloudPlatform/healthcare-dicom-dicomweb-adapter/tree/master/samples)
```
Comment (danielbeaudreau) [inline README.md]: Phrasing suggestion: "The adapters can be deployed as a gke_workload using the [data protection toolkit.](...)"

File: README.md (hunk 1 above)
Comment (pavertomato) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (pavertomato) [reply README.md]: done

**Comment by pavertomato** [comment]
> Thanks!
> 
> I think this would fit more in the Deployment With Kubernetes section rather than Building from Source

done

**Pull request merged**